lib/active_fedora/relation/finder_methods.rb

```diff
@@ -189,17 +189,16 @@ def find_one(id, cast = nil)
     protected
 
       def load_from_fedora(id, cast)
-        raise ActiveFedora::ObjectNotFoundError if id.empty?
+        if id.empty?
```

Comment:
I was trying to keep line length down, but I didn't see that it's disabled in rubocop.